aboutme.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      body {
```

Comment:
instead of style tag, you can have css files and import them

Reply:
Done